kornia/feature/keynet.py

```diff
@@ -186,8 +186,8 @@ def __init__(
         pretrained: bool = False,
         num_features: int = 2048,
         keynet_conf: Dict = keynet_config['KeyNet_default_config'],
-        ori_module: nn.Module = PassLAF(),
-        aff_module: nn.Module = PassLAF(),
+        ori_module: Module = PassLAF(),
```

Comment:
```suggestion
        ori_module: Optional[Module] = None,
```
In the code, if none then PassLaf()

Reply:
Done on [cd80cf1](https://github.com/kornia/kornia/pull/1995/commits/cd80cf17facdf1c2adf6f630b51ee21eb1a1e826)